modules/apps/account/account-service/src/main/java/com/liferay/account/service/impl/AccountGroupLocalServiceImpl.java

```diff
@@ -65,6 +66,24 @@ public AccountGroup addAccountGroup(
 		return accountGroupPersistence.update(accountGroup);
 	}
 
+	@Override
+	public AccountGroup deleteAccountGroup(
+		AccountGroup accountGroup, long accountGroupId) {
```

Comment:
I think you can just get away with only the `accountGroup` here. No need for the `accountGroupId`.